### PR TITLE
Update names and add more service names

### PR DESCRIPTION
**Pull request by v-xuto**
1.Update names and add more service names in donet, python, js and java csv files.

Note: I have updated all. Please Review.

@jongio for notification.

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -843,3 +851,6 @@
 "Microsoft.WindowsAzure.Management.TrafficManager","","0.16.1-preview","WindowsAzure Management - Traffic Manager","Traffic Manager","NA","NA","NA","mgmt","false","",""
 "Microsoft.WindowsAzure.Management.VirtualNetworks","","0.9.9-preview","WindowsAzure Management - Virtual Networks","Virtual Network","NA","NA","NA","mgmt","false","",""
 "Microsoft.WindowsAzure.Management.WebSites","","5.0.0-prerelease","WindowsAzure Management - WebSites","App Service","NA","NA","NA","mgmt","false","",""
+"Microsoft.CognitiveServices.Speech","1.14.0","","Text to Speech","Text to Speech","NA","NA","NA","","client","false",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:854]: Why were these manually added? In general if there are packages not in this list but are published to nuget we need to get azure-sdk added as an owner for those packages and they will automatically get added. 

Also for these entries it looks like are missing at least one field as you have the field "New" set to "client" which isn't correct.

Comment (jongio) [reply _data/releases/latest/dotnet-packages.csv:854]: @weshaggard - It looks like azure-sdk is owner of this package.  Can you see why the script didn't pick it up?

Comment (jongio) [reply _data/releases/latest/dotnet-packages.csv:854]: Microsoft.CognitiveServices.Speech

File: _data/releases/latest/dotnet-packages.csv [hunk 2]
```diff
@@ -843,3 +851,6 @@
...
+"Microsoft.CognitiveServices.Speech","1.14.0","","Speech to Text","Speech to Text","NA","NA","NA","","client","false",""
+"Microsoft.Azure.RemoteRendering","0.1.61","","","Remote Rendering","NA","NA","NA","","client","false",""
```
Comment (jongio) [inline _data/releases/latest/dotnet-packages.csv:856]: @weshaggard azure-sdk not owner here: https://www.nuget.org/packages/Microsoft.Azure.RemoteRendering.Cpp/

File: _data/releases/latest/js-packages.csv [hunk 3]
```diff
@@ -541,10 +548,11 @@
...
+"@azure/cognitiveservices-translatortext","1.0.1","","Translator","Translator","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/cognitiveservices/cognitiveservices-translatortext","NA","NA","client","false","",""
 "@azure/cognitiveservices-videosearch","3.0.1","","Video Search","Video Search","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/cognitiveservices/cognitiveservices-videosearch","NA","NA","client","false","",""
 "@azure/cognitiveservices-visualsearch","3.0.1","","Visual Search","Visual Search","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/cognitiveservices/cognitiveservices-visualsearch","NA","NA","client","false","",""
 "@azure/cognitiveservices-websearch","3.0.1","","Web Search","Web Search","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/cognitiveservices/cognitiveservices-websearch","NA","NA","client","false","",""
+"@microsoft/immersive-reader-sdk","0.5.2","","Immersive Reader","Immersive Reader","https://github.com/microsoft/immersive-reader-sdk","NA","NA","client","false","",""
```
Comment (jongio) [inline _data/releases/latest/js-packages.csv:555]: @weshaggard - This is not a package, just scripts in github, but still a T1 sdk that we provide

**Review by jongio — CHANGES_REQUESTED** [review]

File: _data/releases/latest/dotnet-packages.csv (hunk 1 above)
Comment (weshaggard) [reply _data/releases/latest/dotnet-packages.csv:854]: This one is already in the file. See line 532.

File: _data/releases/latest/js-packages.csv (hunk 3 above)
Comment (weshaggard) [reply _data/releases/latest/js-packages.csv:555]: That seems a little odd. It isn't even in the azure namespace and not on npm but if we need it for tracking purposes I can live with it being added.

File: _data/releases/latest/dotnet-packages.csv [hunk 4]
```diff
@@ -492,7 +498,7 @@
 "Microsoft.Azure.Media.LiveVideoAnalytics.Edge","","1.0.4-preview.1","Media Live Video Analytics Edge","Media Services","https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/mediaservices/Microsoft.Azure.Media.LiveVideoAnalytics.Edge","NA","","client","False","",""
 "Microsoft.Azure.Amqp","2.4.9","","Microsoft.Azure.Amqp","Microsoft.Azure.Amqp","NA","NA","NA","client","False","",""
 "Microsoft.Azure.Devices","1.28.1","","Microsoft.Azure.Devices","Microsoft.Azure.Devices","NA","NA","NA","client","False","",""
-"Microsoft.Azure.Devices.Client","1.33.1","","Microsoft.Azure.Devices.Client","Microsoft.Azure.Devices.Client","NA","NA","NA","client","False","",""
+"Microsoft.Azure.Devices.Client","1.33.1","","Devices Client","IoT Plug and Play","NA","NA","NA","client","False","",""
```
Comment (jongio) [inline _data/releases/latest/dotnet-packages.csv:501]: Are you sure this is for Plug and Play?

File: _data/releases/latest/java-packages.csv [hunk 5]
```diff
@@ -317,9 +322,9 @@
 "iot-deps-preview","com.microsoft.azure.sdk.iot","1.1.0","","IoT Deps","IoT","NA","NA","NA","client","false","true",""
 "iot-deps-serializer","com.microsoft.azure","0.3.3","","IoT Deps Serializer","IoT","NA","NA","NA","client","false","true",""
 "iot-deps-serializer","com.microsoft.azure.sdk.iot.deps","0.7.6","","IoT Deps Serializer","IoT","NA","NA","NA","client","false","",""
-"iot-device-client","com.microsoft.azure","1.1.24","","IoT Device Client","IoT","NA","NA","NA","client","false","true",""
-"iot-device-client","com.microsoft.azure.sdk.iot","1.26.0","1.29.0-preview-001","IoT Device Client","IoT","NA","NA","NA","client","false","",""
-"iot-device-client-preview","com.microsoft.azure.sdk.iot","1.2.0","","IoT Device Client","IoT","NA","NA","NA","client","false","true",""
+"iot-device-client","com.microsoft.azure","1.1.24","","IoT Plug and Play","IoT","NA","NA","NA","client","false","true",""
```
Comment (jongio) [inline _data/releases/latest/java-packages.csv:325]: Why Plug and Play.  I doubt these are PnP

File: _data/releases/latest/dotnet-packages.csv (hunk 4 above)
Comment (v-xuto) [reply _data/releases/latest/dotnet-packages.csv:501]: @jongio - I refer to this link: https://docs.microsoft.com/en-us/azure/iot-pnp/libraries-sdks

File: _data/releases/latest/java-packages.csv (hunk 5 above)
Comment (v-xuto) [reply _data/releases/latest/java-packages.csv:325]: Same as above

File: _data/releases/latest/dotnet-packages.csv [hunk 6]
```diff
@@ -843,3 +894,11 @@
 "Microsoft.WindowsAzure.Management.TrafficManager","","0.16.1-preview","WindowsAzure Management - Traffic Manager","Traffic Manager","NA","NA","NA","mgmt","false","",""
 "Microsoft.WindowsAzure.Management.VirtualNetworks","","0.9.9-preview","WindowsAzure Management - Virtual Networks","Virtual Network","NA","NA","NA","mgmt","false","",""
 "Microsoft.WindowsAzure.Management.WebSites","","5.0.0-prerelease","WindowsAzure Management - WebSites","App Service","NA","NA","NA","mgmt","false","",""
+"Microsoft.Azure.RemoteRendering","0.1.61","","Remote Rendering","Remote Rendering","NA","NA","NA","","client","false",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:897]:
```suggestion
"Microsoft.Azure.RemoteRendering","0.1.61","","Remote Rendering","Remote Rendering","NA","NA","NA","client","false","",""
```
I think a number of these have the fields shifted by 1 and so you should shift them over.

File: _data/releases/latest/dotnet-packages.csv [hunk 7]
```diff
@@ -523,7 +572,8 @@
 "Microsoft.Azure.SignalR.Benchmark","","1.0.0-preview1-10415","SignalR - Benchmark","SignalR","https://github.com/azure/azure-signalr-bench","NA","NA","client","False","",""
 "Microsoft.Azure.SignalR.Protocols","1.6.1","","SignalR - Protocols","SignalR","https://github.com/Azure/azure-signalr/tree/v1.5.0/src/Microsoft.Azure.SignalR.Protocols","NA","NA","client","False","",""
 "Microsoft.Azure.SignalR.Serverless.Protocols","1.2.2","","SignalR - Serverless Protocols","SignalR","https://github.com/Azure/azure-functions-signalrservice-extension/tree/v1.2.0/src/Microsoft.Azure.SignalR.Serverless.Protocols","NA","NA","client","False","",""
-"Microsoft.CognitiveServices.Speech","1.14.0","","Speech","Speech","NA","NA","NA","client","False","",""
+"Microsoft.CognitiveServices.Speech","1.14.0","","Text to Speech","Text to Speech","NA","NA","NA","client","False","",""
+"Microsoft.CognitiveServices.Speech","1.14.0","","Speech to Text","Speech to Text","NA","NA","NA","client","False","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:576]: Why is this in here twice?

File: _data/releases/latest/js-packages.csv [hunk 8]
```diff
@@ -681,3 +733,7 @@
 "@azure/arm-vmwarecloudsimple","1.0.0","","Resource Management - VMware CloudSimple","VMWare by CloudSimple","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/vmwarecloudsimple/arm-vmwarecloudsimple","NA","","mgmt","false","",""
 "@azure/arm-webservices","0.1.1","","Resource Management - Web Services","Machine Learning","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/machinelearning/arm-webservices","NA","","mgmt","false","",""
 "@azure/arm-workspaces","0.1.1","","Resource Management - Work Spaces","Machine Learning","https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/machinelearning/arm-workspaces","NA","","mgmt","false","",""
+"microsoft-cognitiveservices-speech-sdk","1.14.1","","Text to Speech","Text to Speech","NA","https://docs.microsoft.com/javascript/api/overview/azure/speech-service","NA","client","false","",""
+"microsoft-cognitiveservices-speech-sdk","1.14.1","","Speech to Text","Speech to Text","NA","https://docs.microsoft.com/javascript/api/overview/azure/speech-service","NA","client","false","",""
```
Comment (weshaggard) [inline _data/releases/latest/js-packages.csv:737]: Why is this listed twice?

File: _data/releases/latest/python-packages.csv [hunk 9]
```diff
@@ -245,9 +299,11 @@
...
 "uamqp","1.2.13","","Uamqp","Uamqp","https://github.com/Azure/azure-uamqp-python/tree/v1.2.10/uamqp","NA","NA","client","false","",""
 "azure-cognitiveservices-search-videosearch","2.0.0","","Video Search","Video Search","cognitiveservices","NA","","client","false","",""
-"azure-cognitiveservices-vision-face","0.4.1","","Vision Face","Face","cognitiveservices","","","client","false","",""
+"azure-cognitiveservices-vision-face","0.4.1","","Face","Face","cognitiveservices","","","client","false","",""
 "azure-cognitiveservices-search-visualsearch","0.2.0","","Visual Search","Visual Search","https://github.com/Azure/azure-sdk-for-python/tree/azure-cognitiveservices-search-visualsearch_0.2.0/azure-cognitiveservices-search-visualsearch/","NA","","client","false","",""
 "azure-cognitiveservices-search-websearch","2.0.0","","Web Search","Web Search","cognitiveservices","NA","","client","false","",""
+"azure-cognitiveservices-speech","1.14.0","","Text to Speech","Text to Speech","NA","NA","NA","client","false","",""
```
Comment (weshaggard) [inline _data/releases/latest/python-packages.csv:305]: Why is this listed twice?

**Review by weshaggard — APPROVED** [review]
Aside from the duplicate speech SDK libraries the changes look good.

File: _data/releases/latest/dotnet-packages.csv [hunk 10]
```diff
@@ -523,7 +572,8 @@
 "Microsoft.Azure.SignalR.Benchmark","","1.0.0-preview1-10415","SignalR - Benchmark","SignalR","https://github.com/azure/azure-signalr-bench","NA","NA","client","False","",""
 "Microsoft.Azure.SignalR.Protocols","1.6.1","","SignalR - Protocols","SignalR","https://github.com/Azure/azure-signalr/tree/v1.5.0/src/Microsoft.Azure.SignalR.Protocols","NA","NA","client","False","",""
 "Microsoft.Azure.SignalR.Serverless.Protocols","1.2.2","","SignalR - Serverless Protocols","SignalR","https://github.com/Azure/azure-functions-signalrservice-extension/tree/v1.2.0/src/Microsoft.Azure.SignalR.Serverless.Protocols","NA","NA","client","False","",""
-"Microsoft.CognitiveServices.Speech","1.14.0","","Speech","Speech","NA","NA","NA","client","False","",""
+"Microsoft.CognitiveServices.Speech","1.14.0","","Text to Speech","Text to Speech","NA","NA","NA","client","False","",""
```
Comment (jongio) [inline _data/releases/latest/dotnet-packages.csv:575]: Please don't include sub-services like this in the CSVs.  Thanks. Only include the primary service.  The sub-services are needed for inventory, but we can manually add those.

**Review by jongio — APPROVED** [review]

**Pull request merged**